content/applications/hr/payroll.rst

```diff
@@ -238,29 +368,38 @@ The required fields for a rule are:
   Code`. Depending on what is selected, the fixed amount, percentage, or Python code needs to be
   entered next.
 
+.. image:: payroll/new-rule.png
+   :align: center
+   :alt: Enter the information for the new rule on the new rule form.
+
 Rule parameters
 ---------------
 
 .. note::
    Currently, the :guilabel:`Rule Parameters` feature found inside the :menuselection:`Payroll app
-   --> Configuration` menu is still in development and only serves a specific use case for Belgian
-   markets. The documentation will be updated when this section has matured to more markets.
+   --> Configuration --> Salary --> Rule Parameters` menu is still in development and only serves a
+   specific use case for Belgian markets. The documentation will be updated when this section has
+   matured to more markets.
 
 Other input types
 -----------------
 
 When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
 like expenses, reimbursements, or deductions. These other inputs can be configured by going to
-:menuselection:`Payroll --> Configuration --> Other Input Types`.
+:menuselection:`Payroll --> Configuration --> Salary --> Other Input Types`.
 
 .. image:: payroll/other-input.png
    :align: center
-   :alt: Other input types for payroll.
+   :alt: A list of other input types for payroll tha can be selected when creating a new entry for a
+         payslip.
 
-To create a new input type, click the :guilabel:`Create` button. Enter the :guilabel:`Description`,
-the :guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
-field. Click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
-the entry.
+To create a new input type, click the :guilabel:`New` button. Enter the :guilabel:`Description`, the
+:guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
+field.
+
+The :guilabel:`Code` is used in the salary rules to compute payslips. If the :guilabel:`Availability
+in Structure` field is left blank, it indicates that the new input type is available for all
+payslips and is not exclusive to a specific structure.
```

Comment:
shift sentence about availability in structure to above, or make as a important/note block

Reply:
Made it an Important block.